server/neptune/gateway/event/check_run_handler_test.go

```diff
@@ -3,20 +3,23 @@ package event_test
 import (
```

Comment:
Would be nice to have some tests for `rerequested` action type that asserts:
- block force applies
- skip if platform mode not enabled for root